test/events-reader.spec.js

```diff
@@ -151,7 +147,9 @@ describe('onChange callback, event capture and at-least-once delivery semantics'
             ]);
           })
           .then(function() {
-            return initFromLastCheckpoint(harvesterApp, oplogDb);
+            return initFromLastCheckpoint(harvesterApp);
+          }).catch(err => {
+            console.log(err);
```

Comment:
This is rarely the right way to handle an error. I dont think it's the right where here either.